api_contracts/contracts/award/AwardProfile.md

```diff
@@ -123,29 +189,28 @@ This endpoint returns a list of data that is associated with the award profile p
 
 ## Recipient (object)
 + recipient_name: `Booz Allen Hamilton` (required, string, nullable)
-+ recipient_unique_id: 2424224 (required, number, nullable)
-+ parent_recipient_unique_id: 2424232 (required, number, nullable)
++ recipient_hash: `e4096343-5e8f-352a-f8af-d2a8b0f0ae68-C` (required, string)
++ recipient_unique_id: `2424224` (required, string, nullable)
+    The recipient's DUNS
++ parent_recipient_unique_id: `2424232` (required, string, nullable)
+    The recipient's parent's DUNS
 + location (required, Location, nullable)
-+ recipient_parent_name: `HoneyWell` (string, nullable)
-+ business_categories_name (required, array[string])
-    Name of business catagories in human readable format
+    The recipeint's location
++ parent_recipient_name: `HoneyWell` (string, nullable)
++ business_categories (required, array[string])
+    Names of the recipients' business catagories in human readable format
 
-## PerformancePeriod
-+ period_of_performance_start_date: `2004-02-19` (required, string)
-    The starting date of the contract in the format `YYYY-MM-DD`
-+ period_of_performance_current_end_date: `2005-02-19` (required, string)
-    The ending date of the contract in the format `YYYY-MM-DD`
-+ action_date: `2301-01-20` (required, string)
-    The date the award was awarded on
+##PeriodOfPerformance
++ start_date: `2004-02-19` (required, string)
++ end_date: `2005-02-19` (required, string)
+    Corresponds to database fields current_end_date for contracts and ordering_period_end_date for IDVs
 + last_modified_date: `2301-02-20` (required, string)
-+ potential_end_date: `2301-02-23` (required, string)
-+ ordering_period_end_date: `2301-02-23` (required, string, nullable)
++ potential_end_date: `2301-02-23` (required, string, nullable)
 
-## IdvDates
+##PeriodOfPerformanceAssistance
 + start_date: `2004-02-19` (required, string)
```

Comment:
I need to investigate if these fields for Assistance-types can be null, first glance is "yes" (Different than FPDS which appear to be required in upstream systems)